### PR TITLE
Node service updates and fixes

### PR DESCRIPTION
**Pull request by salman-codewar**
PR addressed:

- Moved node connectivity and state enums to common
- Removed unnecessary rpc's

### CODE REVIEW

File: systems/common/ukama/node_state.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import (
+	"database/sql/driver"
+	"strconv"
+	"strings"
+)
+
+type NodeState uint8
+
+const (
+	Undefined NodeState = iota
```
Comment (Brackleycassinga) [inline systems/common/ukama/node_state.go:20]: We have changed these states to :
unknown, config, operational ,faulty

File: systems/registry/node/pkg/db/model.go [hunk 2]
```diff
@@ -151,3 +62,20 @@ func (s *Site) BeforeSave(tx *gorm.DB) (err error) {
 
 	return nil
 }
+
+func ParseNodeState(s string) ukama.NodeState {
```
Comment (Brackleycassinga) [inline systems/registry/node/pkg/db/model.go:66]: This parsing func should also be part of common i guess.

File: systems/registry/node/pkg/config.go [hunk 3]
```diff
@@ -62,6 +62,7 @@ func NewConfig(name string) *Config {
 		},
 		Service: uconf.LoadServiceHostConfig(name),
 		MsgClient: &uconf.MsgClient{
+			// Host:    "msgclient-registry:9095",
```
Comment (Brackleycassinga) [inline systems/registry/node/pkg/config.go:65]: This msg client host comment can be removed.

File: systems/registry/node/pkg/db/model.go (hunk 2 above)
Comment (salman-codewar) [reply systems/registry/node/pkg/db/model.go]: addressed

File: systems/registry/node/pkg/config.go (hunk 3 above)
Comment (salman-codewar) [reply systems/registry/node/pkg/config.go]: addressed

**Review by Brackleycassinga — APPROVED** [review]

**Pull request merged**